src/MooLite/plugins/EquipmentExporter/EquipmentExporterPluginDisplay.vue

```diff
@@ -0,0 +1,162 @@
+          equipment["/equipment_types/hands"].hrid = equipmentData[i].name;
+          equipment["/equipment_types/hands"].enhancementLevel = equipmentData[i].level;
+        } else if(equipmentData[i].name.includes("_boots")){
+          equipment["/equipment_types/feet"].hrid = equipmentData[i].name;
+          equipment["/equipment_types/feet"].enhancementLevel = equipmentData[i].level;
+        } else if(equipmentData[i].name.includes("_pouch")){
+          equipment["/equipment_types/pouch"].hrid = equipmentData[i].name;
+          equipment["/equipment_types/pouch"].enhancementLevel = equipmentData[i].level;
+        } else if(equipmentData[i].name.includes("necklace")){
+          equipment["/equipment_types/neck"].hrid = equipmentData[i].name;
+          equipment["/equipment_types/neck"].enhancementLevel = equipmentData[i].level;
+        } else if(equipmentData[i].name.includes("earrings")){
+          equipment["/equipment_types/earrings"].hrid = equipmentData[i].name;
+          equipment["/equipment_types/earrings"].enhancementLevel = equipmentData[i].level;
+        } else if(equipmentData[i].name.includes("ring_")){
+          equipment["/equipment_types/ring"].hrid = equipmentData[i].name;
+          equipment["/equipment_types/ring"].enhancementLevel = equipmentData[i].level;
+        } else if(equipmentData[i].name.includes("_sword") || equipmentData[i].name.includes("_spear") || equipmentData[i].name.includes("_mace") || equipmentData[i].name.includes("_bludgeon")){
+          equipment["/equipment_types/main_hand"].hrid = equipmentData[i].name;
+          equipment["/equipment_types/main_hand"].enhancementLevel = equipmentData[i].level;
+        } else if(equipmentData[i].name.includes("_bulwark") || equipmentData[i].name.includes("_slasher")){
+          equipment["/equipment_types/main_hand"].hrid = equipmentData[i].name;
+          equipment["/equipment_types/main_hand"].enhancementLevel = equipmentData[i].level;
+        } else if(equipmentData[i].name.includes("_buckler") || equipmentData[i].name.includes("_dirk") || equipmentData[i].name.includes("_shield")) {
+          equipment["/equipment_types/off_hand"].hrid = equipmentData[i].name;
+          equipment["/equipment_types/off_hand"].enhancementLevel = equipmentData[i].level;
+        } else if(equipmentData[i].name.includes("coffee")) {
+          drinks.push(equipmentData[i].name);
+        } else if(equipmentData[i].name.includes("donut") || equipmentData[i].name.includes("cake") || equipmentData[i].name.includes("gummy") || equipmentData[i].name.includes("yogurt")) {
+          food.push(equipmentData[i].name);
+        }
+      }
+    }
+    for (let equip in equipment){
+      if(equipment[equip].hrid === ""){
+        equipment[equip] = null;
+      }
+    }
+    while(drinks.length < 3){
+      drinks.push("");
+    }
+    while(food.length < 3){
+      food.push("");
+    }
```

Comment:
I'd prefer not to use while loops for the chance of infinite loops. Also we can get the max length from game data